technology/web/src/main/java/com/github/mizool/technology/web/healthcheck/Check.java

```diff
@@ -0,0 +1,6 @@
+package com.github.mizool.technology.web.healthcheck;
```

Comment:
add license boilerplate comments to all new java files

Reply:
Can we _please_ change the license already? *growls*

If I knew how, I would have already done that.